### ISSUE TITLE
Audio on iOS changing from headphones to phone speakers

### ISSUE THREAD

**Issue by migueloli** [rive-app/rive-flutter#416]
### Description
When the `.riv` file has an audio on iOS we are getting the audio sent to the device speakers and also changing the audio you had playing to the speaker.

e.g.: I was testing while listening some songs on Spotify, and when I got the animation the audio for the animations played on my device speaker and the Spotify started to play on speaker from that point on.

### Steps To Reproduce
`
  RiveAnimation.direct(
    riveFile,
    onInit: _onInitRiveAnimation,
    useArtboardSize: true,
  );
`
`
  void _onInitRiveAnimation(Artboard artboard) {
    final controller = StateMachineController.fromArtboard(
      artboard,
      'State Machine 1',
    );

    if (controller != null) {
      final input = controller.getNumberInput('percentage');
      artboard.addController(controller);
      input?.value = _animationController.value;
      _animationController.forward();
      _animationController
          .addListener(() => input?.value = _animationController.value);
    }
  }
`

#### Source `.riv`/`.rev` file
[xp-progress-bar.zip](https://github.com/user-attachments/files/16758399/xp-progress-bar.zip)

### Expected behavior
We would expect to play the audio from the headphones when they are connected, and not force the audio to be played from the device speaker.

### Device & Versions
Tested with another iOS devices, but last test was using this one:
- Device: iPhone 11
- OS: iOS 17.6.1

```
Flutter 3.24.0 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 80c2e84975 (4 weeks ago) • 2024-07-30 23:06:49 +0700
Engine • revision b8800d88be
Tools • Dart 3.5.0 • DevTools 2.37.2
```

#### Additional context
Android is working as expected on our tests.

Thank you!

**Comment by HayesGordon** [comment]
Hi @migueloli, thank you for reporting the issue. We already have a fix for this on our native integration, and once that lands we can update the Flutter runtime to include it.

**Comment by migueloli** [comment]
Thank you @HayesGordon

**Comment by HayesGordon** [comment]
This should now be resolved in v0.13.13

Please reopen if you're still experiencing issues after upgrading.
